webapp/src/codecard.tsx

```diff
@@ -16,11 +17,7 @@ export class CodeCardView extends React.Component<pxt.CodeCard, CodeCardState> {
         this.state = {};
     }
 
-    componentDidUpdate() {
-        ($('.ui.embed') as any).embed();
```

Comment:
Do we support YouTube video embeds?

Reply:
Not sure, can you give an example?

Reply:
(also this is for codecard.tsx which is the react equivalent of the docrender one) and only used in the editor. I don't believe we support youtube embeds in the editor.

Reply:
It’s used in docs but I don’t think we load the YouTube iframe in the editor.

Reply:
Cool. I think it's fine to get rid of it then.